deployer/src/config_creator.py

```diff
@@ -33,7 +33,14 @@ def extract_root_from_input(input_string):
         return domain.group() if domain else input_string
 
 
+def is_non_empty_list(is_non_empty_list):
+    # Check if the input is a non-empty list. Otherwise, it raises an exception
+    if not is_non_empty_list or len(is_non_empty_list) == 0:
```

Comment:
If the input list is None (which is the default in callers), the exception will get raised; is it expected?

Reply:
Yes this is expected, we want to make sure the input is not None or empty after calling this function. Comment updated. Thanks for reviewing